src/core/server/opensearch/status.ts

```diff
@@ -77,7 +77,7 @@ export const calculateStatus$ = (
 
           return {
             level: ServiceStatusLevels.available,
-            summary: `OpenSearch is available`,
+            summary: message ?? `OpenSearch is available`,
```

Comment:
```suggestion
            summary: message || `OpenSearch is available`,
```
`||` is a safer operator to guard against `''` and `false`.

Reply:
Though, i would prefer to be stricter..
```suggestion
            summary: (message ?? `OpenSearch is available`) || `Unknown`,
```

Reply:
Good idea! Refactored to be more stricter.